### PR TITLE
chore: Add message with the values of Wizeline

### LINKED ISSUE #128: Add message from POP's.
Add the following message in the homepage. 

> Hello, {{firstname}}!
> Welcome to Wize Q! We want to share a few simple guidelines before you start.
> Remember that Wize Q is a space for asking questions and providing answers that are helpful to our community.
> Please practice our values when using Wize Q:
>  - **Ownership** – See if you can find the answer to your question before posting on Wize Q. And if you know the answer to a question or how to find it, be sure to reply — anyone can!
>  - **Innovation** – When someone shares a concern or challenge, let’s be innovative — propose a solution or offer support!
> - **Community** – Remember to treat everyone with dignity and respect. Assume others have good intentions. Always be honest and constructive. Let’s make Wizeline a community where everyone can thrive.
> 
>Consider that other channels (ticketing portal, Slack, or your manager or people partner) might be more effective for finding the right answer quickly.
> Thanks for being a valuable contributor to our community!
> The Wize Q Team

### PR DESCRIPTION
**Pull request by GeorginaFraire**
#### What does this PR do?
- Add an alert to show the message with the values of Wizeline, When the user log in. 

#### Where should the reviewer start?
-  app/components/ListQuestions/ListQuestions.jsx 
-  app/components/Login/LoginContainer.jsx
-  app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx
-  app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx
-  app/utils/cookies.js

#### How should this be manually tested?
1. Login
2. Verify that the message appears, then close it.
3. Go to another page and go to the home page and verify the message not appears. 
4. Logout 
5. Login and verify that the message appears again.

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #128

#### Screenshots

https://user-images.githubusercontent.com/97203617/207971638-b5b0c3d1-0521-49cb-bbc1-863228211d58.mov

### CODE REVIEW

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  const profile = useUser();
+
+  const renderBulletPoint = (color) => (
+    <BsCircleFill color={color} size="7px" style={{ marginTop: '3px', marginRight: '10px' }} />
+  );
+
+  if (!show) { return null; }
+  return (
+    <div onClick={onClose}>
+      <styled.Modal onClicl={onClose}>
```
Comment (Hoosep) [inline app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx:19]: I think your have a little typo here.
```suggestion
      <styled.Modal onClick={onClose}>
```

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+export const ModalFooter = styled.div`
+    border-top: none;
+    background-color: #fff;
+    border-bottom: none;
+    padding: 15px;
+    text-align: right;
+    box-sizing: border-box;
+    display: block;
+    font-size: 14px;
+    ${(props) => (props.variant === 'logout' ? 'border-top: 1px solid #e5e5e5;padding: 15px;text-align: right;' : 'border-bottom: none;')}
```
Comment (Hoosep) [inline app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx:62]:
```suggestion
    ${(props) => (props.variant === 'logout' ? `
       border-top: 1px solid #e5e5e5;
       padding: 15px;
       text-align: right;
    ` : 'border-bottom: none;')}
```

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+`;
+
+export const ModalHeader = styled.div`
+    overflow-y: hidden;
+    padding: 15px;
+    box-sizing: border-box;
+    display: block;
+    font-size: 14px;
+    ${(props) => (props.variant === 'logout' ? 'border-bottom: 1px solid #e5e5e5;padding: 15px;' : 'border-bottom: none;')}
```
Comment (Hoosep) [inline app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx:71]:
```suggestion
    ${(props) => (props.variant === 'logout' ? `
       border-bottom: 1px solid #e5e5e5;
       padding: 15px;
       ` : 'border-bottom: none;')}
```

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+export const ModalSubtitle = styled.div`
+    font-size: 16px;
+    font-weight: normal;
+    letter-spacing: 0.6px;
+    line-height: 1.71;
+`;
+
+export const ValueText = styled.span`
+color: ${(props) => props.color};
```
Comment (Hoosep) [inline app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx:90]: Add space.

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+font-weight: 600;
+font-size: 16px;
+text-decoration:  ${(props) => props.color} underline overline;
+`;
+
+export const ValuesInformation = styled.div`
+margin: 20px 0 20px 20px;
```
Comment (Hoosep) [inline app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx:97]: Add space.

File: app/components/ListQuestions/ListQuestions.jsx [hunk 6]
```diff
@@ -27,8 +30,16 @@ function ListQuestions({
   // TODO: Implement search
   const state = {
     searchTerm: undefined,
+    showValuesMessage: true,
   };
 
+  const valuesMessageModal = showValuesMessage === 'true' ? (
```
Comment (Hoosep) [inline app/components/ListQuestions/ListQuestions.jsx:36]:
```suggestion
  const valuesMessageModal = showValuesMessage === 'true' && (
```

File: app/components/ListQuestions/ListQuestions.jsx [hunk 7]
```diff
@@ -27,8 +30,16 @@ function ListQuestions({
   // TODO: Implement search
   const state = {
     searchTerm: undefined,
+    showValuesMessage: true,
```
Comment (Hoosep) [inline app/components/ListQuestions/ListQuestions.jsx:33]: This is useful? I see you aren't using it in your code.

File: app/components/ListQuestions/ListQuestions.jsx [hunk 8]
```diff
@@ -27,8 +30,16 @@ function ListQuestions({
...
   };
 
+  const valuesMessageModal = showValuesMessage === 'true' ? (
+    <ValuesMessageModal
+      show={showValuesMessage}
+      onClose={() => { setShowValueMessage(false); setCookie('showValueMessage', false); }}
+    />
+  ) : null;
```
Comment (Hoosep) [inline app/components/ListQuestions/ListQuestions.jsx:41]:
```suggestion
  );
```

**Review by Hoosep — APPROVED** [review]
Leave you some comments but your PR is already approved.

Thank you so much for everything you do with WizeQ, folks! 🎉

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx (hunk 1 above)
Comment (GeorginaFraire) [reply app/components/Modals/ValuesMessageModal/ValuesMessageModal.jsx]: Done

File: app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx (hunk 4 above)
Comment (GeorginaFraire) [reply app/components/Modals/ValuesMessageModal/ValuesMessageModal.Styled.jsx]: Done

File: app/components/ListQuestions/ListQuestions.jsx (hunk 6 above)
Comment (GeorginaFraire) [reply app/components/ListQuestions/ListQuestions.jsx]: Done

**Comment by GeorginaFraire** [comment]
@Hoosep Can you please review it ? Thank you 😄

**Comment by Hoosep** [comment]
> @Hoosep Can you please review it ? Thank you 😄  

It was already approved! LGTM! 🎉✨

**Review by Hoosep — APPROVED** [review]

**Pull request merged**